otelcollector/metricextension/me.config

```diff
@@ -11,9 +11,9 @@
       "honorResourceAttributes":true,
       "maxReceiveMessageSizeMBytes": 12
    },
-   "publicationIntervalInSec":30,
+   "publicationIntervalInSec":20,
    "maxPublicationAttemptsPerMinute":12,
-   "maxPublicationPackageSizeInBytes":100000000,
+   "maxPublicationPackageSizeInBytes":2000000,
```

Comment:
any impact on the scale tests, Grace for moving to 2mb from 100mb ?

Reply:
Things look good with this combined with decreasing the publication interval from 30s to 20s as Emre recommended. For the image from this PR vs our last scale testing, the perf has improved with the same benchmarks for scraping all default targets on a large cluster: 1.7 Gi before vs 1.4 Gi now and 1.1 cores vs. 500mc now for replicaset and confirmed it's the same total amount of metrics are being sent

Reply:
thanks grace